packages/edit-site/src/store/selectors.js

```diff
@@ -10,10 +10,14 @@ import { store as coreDataStore } from '@wordpress/core-data';
 import { createRegistrySelector } from '@wordpress/data';
 import deprecated from '@wordpress/deprecated';
 import { uploadMedia } from '@wordpress/media-utils';
-import { isTemplatePart } from '@wordpress/blocks';
 import { Platform } from '@wordpress/element';
 import { store as preferencesStore } from '@wordpress/preferences';
 
+/**
+ * Internal dependencies
+ */
+import { memoizedGetFilteredTemplatePartBlocks } from './utils';
```

Comment:
I guess we're forced to use another file because all exported functions here are selectors?